doc/nrf/includes/thread_configure_network.txt

```diff
@@ -8,5 +8,5 @@ The following example uses the default OpenThread parameters:
    Done
    uart:~$ ot panid 0xabcd
    Done
-   uart:~$ ot masterkey 00112233445566778899aabbccddeeff
+   uart:~$ ot networkkey 00112233445566778899aabbccddeeff
```

Comment:
there is also masterkey in line 1: ...``ot channel``, ``ot panid``, and ``ot **masterkey**`` commands.